From 6cad1f12f6b4c42c7a6c600c056f79617b30b6cd Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Wed, 20 Dec 2023 17:42:49 +0200
Subject: [PATCH] [SW-0] gaudi3: pif: fix flr active reg addr

During transition to new specs scheme a wrong register was assigned
to the flr_active register of the PIF ARC

Change-Id: I532def3f1fe0747f74eeb078681c539c295fa490
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 349d8c7..865fb0d 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -594,7 +594,7 @@
 
 void device_init_pif_arc(void)
 {
-	pif_arc.pf_flr_active = D0_PIF_ARC_REG(mmPIF_ARC_PF_FLR_DONE);
+	pif_arc.pf_flr_active = D0_PIF_ARC_REG(mmPIF_ARC_PF_FLR_ACTIVE);
 	pif_arc.pf_flr_done = D0_PIF_ARC_REG(mmPIF_ARC_PF_FLR_DONE);
 	pif_arc.inbound_term = D0_PIF_ARC_REG(mmPIF_ARC_INBOUND_TERM);
 	pif_arc.crs = D0_PIF_ARC_REG(mmPIF_ARC_CRS);
